hazelcast/src/main/java/com/hazelcast/jet/impl/client/protocol/task/JetGetJobAndSqlSummaryListMessageTask.java

```diff
@@ -60,7 +60,9 @@ public Collection<Member> getTargets() {
     @SuppressWarnings("unchecked")
     @Override
     protected Object reduce(Map<Member, Object> map) throws Throwable {
+        rethrowInReduce(map, true);
         return map.values().stream()
+                .filter(item -> item instanceof List<?>)
```

Comment:
Do we need this? Now we assert that all items will be `List` instances. It might hide problems.

Reply:
This is needed, since you can have both ```MemberLeftException``` and real result in the map.

Reply:
I see. Then it would be better if it's this:
```suggestion
                .filter(item -> !(item instanceof MemberLeftException) || !(item instanceof TargetNotMemberException))
```

Reply:
Or maybe `item -> !(item instanceof Throwable)` as in the line above we are throwing for any other exception type

Reply:
Fixed with @mdumandag approach.